lib/util/init.js

```diff
@@ -16,39 +16,39 @@ const Init = {
     /**
      * Creates template file for properties.json
      * @param {Object} properties config file's json
-     * @param {string} credentialsName identifying name of the installed package / project
+     * @param {string} credentialName identifying name of the installed package / project
      * @param {Object} [skipInteraction] signals what to insert automatically for things usually asked via wizard
-     * @param {String} skipInteraction.clientId client id of installed package
-     * @param {String} skipInteraction.clientSecret client secret of installed package
-     * @param {String} skipInteraction.tenant tenant of installed package
-     * @param {String} skipInteraction.eid MID of the Parent Business Unit
-     * @param {String} skipInteraction.credentialsName how you would like the credential to be named
+     * @param {String} skipInteraction.client_id client id of installed package
+     * @param {String} skipInteraction.client_secret client secret of installed package
+     * @param {String} skipInteraction.auth_rul tenant of installed package
+     * @param {String} skipInteraction.account_url MID of the Parent Business Unit
```

Comment:
why rename?

Reply:
because we should align values, where possible, with the ones being used. otherwise we mix up when clientId and when client_id. This standardizes the values. I would note this is a breaking change ;)